spec/unit/lib/temporal/testing/temporal_override.rb

```diff
@@ -88,7 +88,7 @@ def execute; end
         expect(Temporal::Testing::ScheduledWorkflows.cron_schedules).to be_empty
       end
     end
-    
+
```

Comment:
nit: This is a spec file and needs `_spec.rb` in the filename

Reply:
I'll fix all of these test nits up. This is my first time working with rspec. I definitely appreciate getting these details right.

Reply:
No worries, Jeff 🙌